### ISSUE TITLE
Menubar at top of webpage is spam and should not be visible when reading while scrolled down on page

### ISSUE THREAD

**Issue by seronis** [godotengine/godot-website#33]
Vertical screen space is a limited and precious commodity.   And it is being wrongfully used by a pointless menubar that refuses to stay out of sight at the top of the page where it belongs.

If the menubar was statically placed you can still access it at any time 'instantly'.  Pressing the Home key scrolls a page to the top.  Even on laptops that dont have a dedicated home key you still have the 'function' key that makes one of the other keys act as the home key which takes the exact same amount of time to type. It also takes less than a second to just drag the scroll bar to the top. On a phone (at least on my Android) a couple brisk swipes can scroll to the top of a webpage even if you were 30 'pages' of content down.

In other words, THERE IS NO EXCUSE OR REASON AT ALL to be wasting screen viewing space for something that is still accessible effectively 'instantly' on any platform.

**Comment by timschellenberg** [comment]
That's a very outdated design guideline, especially for presentation-focused web pages. I would totally understand the complaint if godotengine.org was a page that you would power navigate for long periods at a time (like a message board or news site), but its entire focus is to look appealing and be convenient to navigate, it's not very concerned with conserving vertical space.

The big game screenshot taking up the majority of the vertical fold (eg. the portion of the site that is visible immediately upon loading) seems like a bigger concern to me, because on smaller resolutions there just isn't much of any information about what the site is actually about.

**Comment by seronis** [comment]
No, the entire point is to be convenient to READ.   The bar being static is still equally convenient to navigate since its still instantly accessible no matter what.   But being static would mean its no longer IN THE WAY.   Stuff that is IN THE WAY doesnt look good.  It looks spammy.

**Comment by timschellenberg** [comment]
I found it pretty convenient when I was reading the 3.0 release post and wanted to go to the download page without losing where I was in that long article, personally.
Of course it can't be perfect for everyone, but "spammy" or "in the way" is definitely not what I would call a 64 pixel high, often helpful menu bar.

Being instantly accessible isn't really a great argument because if you press the up or down arrow key or move the scroll wheel, the info that the bar is covering up becomes visible too, although the chances that you want to look exactly there are rather low because you'd have to scroll a LOT to cover the giant symbols on the front page completely.

Not to mention a lot of people aren't even aware that the home key does anything.


I totally understand where you're coming from personally, but I think you're overplaying it a lot. Since you seem to be a power user and tech savvy, maybe it'd be worthwhile to just apply a bit of CSS to the `header` tag to keep it static on your end. At least that way if this isn't agreed with by other people it'd be fixed for you (I'm not an authority on this by any measure).

**Comment by LinuxUserGD** [comment]
Blender hides the menu bar when you scroll down which makes it more comfortable to read the page on mobile devices. What about this design? https://www.blender.org/

**Comment by Calinou** [comment]
The navbar shouldn't be fixed, as it really harms the browsing experience on small displays (especially 1366×768 and 1600×900, which are still *very* common today). Today's not-very-space-efficient browsers make the issue even worse (such as Firefox with the default settings).
